javascript-data-model/order-history.js

```diff
@@ -0,0 +1,76 @@
+var orderHistory = [
+  {
+    orderDate: 'August 4, 2020',
+    ifDelivered: true,
+    deliveryDate: 'August 8,2020',
+    subtotal: 31.55,
+    grandTotal: 34,
+    recipient: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    returnWindowOpen: false,
+    returnWindowDate: 'Sep 7, 2020',
+    reviewWritten: false,
+    itemInfo: {
+      type: 'book',
+      name: 'Javascript for impatient programmers',
+      author: 'Dr. Axel Rauschmayer'
+    }
+  },
+  {
+    orderDate: 'July 19, 2020',
+    ifDelivered: true,
+    deliveryDate: 'July 20,2020',
+    subtotal: 41.33,
+    grandTotal: 44.53,
+    recipient: 'JS Masher',
+    orderNumber: '113-9984268-1280257',
+    returnWindowOpen: false,
+    returnWindowDate: 'Aug 19, 2020',
+    reviewWritten: false,
+    itemInfo: {
+      type: 'book',
+      name: 'The Timeless Way of Building',
+      author: 'Christopher Alexander'
+    }
+  },
+  {
+    orderDate: 'July 4, 2020',
+    ifDelivered: true,
+    deliveryDate: 'July 7,2020',
+    subtotal: 15.98,
+    grandTotal: 17.22,
+    recipient: 'JS Masher',
+    orderNumber: '114-2875557-9059409',
+    returnWindowOpen: false,
+    returnWindowDate: 'Aug 5, 2020',
+    reviewWritten: false,
+    itemInfo: {
+      type: 'electronic',
+      name: 'Gamecube Controller Adapter'
+    }
+  },
+  {
+    orderDate: 'July 3, 2020',
+    ifDelivered: true,
+    deliveryDate: 'July 5,2020',
+    returnWindowOpen: false,
+    returnWindowDate: 'Aug 4, 2020',
+    reviewWritten: false,
+    grandTotal: 138.93,
+    recipient: 'JS Masher',
+    orderNumber: '113-2883177-2648248',
+    itemInfo: [
```

Comment:
For this order, the value of the `itemInfo` property is an array, but for all the other orders, it is an object. You should make all the other orders have an array as well, which is holding that object that you currently have set as their value within it.